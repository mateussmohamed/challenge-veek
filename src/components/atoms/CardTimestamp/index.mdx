---
name: CardTimestamp
menu: Atoms
---

import { Playground, PropsTable } from 'docz'
import CardTimestamp from './index'

# CardTimestamp
<PropsTable of={CardTimestamp} />

<Playground>
  <CardTimestamp />
</Playground>
