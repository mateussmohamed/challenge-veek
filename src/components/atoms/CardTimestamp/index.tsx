import styled from 'styled-components'

export const CardTimestamp = styled.div`
  margin-left: 5px;

  p {
    color: #848484;
    font-family: Lato, sans-serif;
    font-size: 14px;
    font-weight: 400;
    text-transform: uppercase;
    line-height: normal;
    margin: 0;
  }
`
