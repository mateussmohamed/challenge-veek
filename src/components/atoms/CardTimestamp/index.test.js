import React from 'react'
import { shallow } from 'enzyme'
import 'jest-styled-components'

import CardTimestamp from './index'

describe('[<CardTimestamp />]', () => {
  const wrap = (props = {}) => shallow(<CardTimestamp {...props} />)

  test('matches to snapshot', () => {
    const wrapper = wrap()

    expect(wrapper).toMatchSnapshot()
  })
})
