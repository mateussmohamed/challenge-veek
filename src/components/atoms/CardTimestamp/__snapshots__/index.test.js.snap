// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[<CardTimestamp />] matches to snapshot 1`] = `
<StyledComponent
  forwardedClass={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": undefined,
      "componentStyle": ComponentStyle {
        "componentId": "CardTimestamp-sc-14nakfg-0",
        "isStatic": true,
        "rules": Array [
          "width:100%;height:100vh;flex-direction:column;@media only screen and (min-width:0em){padding:20px 0;}@media only screen and (min-width:48em){padding:200px 0;}",
        ],
      },
      "displayName": "CardTimestamp",
      "render": [Function],
      "styledComponentId": "CardTimestamp-sc-14nakfg-0",
      "target": "section",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
>
  I am a container!
</StyledComponent>
`;
